autoscheduler/frontend/src/components/Schedule.css

```diff
@@ -62,8 +65,20 @@
 
 .calendar-body {
     position: relative;
+    flex-grow: 1;
+    display: flex;
+    flex-direction: column;
 }
 
 .calendar-day {
     flex-grow: 1;
+}
+
+/* WARNING, may cause problems later */
```

Comment:
This is probably the worst part. I will probably move these rules up into a new module for `App.css`, but the fact remains that all other components will have to deal with this `height: 100%;`. I'd like someone to confirm that we have no plans for any route/component that needs to be larger than the window before making this official.

Reply:
Me and Gannon just discussed this just to make sure we agree, and yes we're trying to fit everything to a single page height, so this shouldn't cause any problems.

Reply:
Resolved in latest commit